zorka-core/src/main/java/com/jitlogic/zorka/core/integ/netuitive/JvmSystemStatsReport.java

```diff
@@ -12,9 +12,11 @@
 
     /* Logger */
     private final ZorkaLog log = ZorkaLogger.getLog(JvmSystemStatsReport.class);
+    private Java6CpuUsage java6CpuUsage;
```

Comment:
My personal preference is the opposite, but this construction is inconsistent with the line right next to it, so I think it's worth changing for consistency with the rest of the project.